src/components/appBar/AppBar.tsx

```diff
@@ -26,20 +37,51 @@ export const AppBar = ({ text, onPress, iconName = 'arrow-left', style, children
         appBar: {
             backgroundColor: theme.sw.colors.neutral[50],
         },
+        container: {
+            flexDirection: 'row',
+            justifyContent: 'center',
+        },
+        touchableOpacity: {
+            paddingHorizontal: theme.sw.spacing.l,
+            paddingVertical: theme.sw.spacing.m,
+            borderRadius: 8,
+            flexDirection: 'row',
+            justifyContent: 'center',
+            alignItems: 'center',
+            backgroundColor: '#ECF0F4',
+            ...style,
+        },
+        body: {
+            fontWeight: '700',
+            paddingRight: theme.sw.spacing.m,
+        },
     });
-    return (
-        <View style={styles.header}>
-            <BaseAppBar style={styles.appBar}>
-                <BaseAppBar.Action
-                    style={styles.appBarStyle}
-                    color={theme.sw.colors.neutral[800]}
-                    size={theme.sw.spacing.l}
-                    icon={iconName}
-                    onPress={onPress}
-                />
-                <Headline size="h3">{text}</Headline>
-            </BaseAppBar>
-            {children}
-        </View>
-    );
+
+    if (type === 'default') {
+        return (
+            <View style={styles.header}>
+                <BaseAppBar style={styles.appBar}>
+                    <BaseAppBar.Action
+                        style={styles.appBarStyle}
+                        color={theme.sw.colors.neutral[800]}
+                        size={theme.sw.spacing.l}
+                        icon={iconName}
+                        onPress={onPress}
+                    />
+                    <Headline size="h3">{label}</Headline>
+                </BaseAppBar>
+                {children}
+            </View>
+        );
+    } else {
+        return (
+            <View style={styles.container}>
+                <TouchableOpacity onPress={onPress} style={styles.touchableOpacity}>
+                    <Body style={styles.body}>{label}</Body>
```

Comment:
I think the Design system uses a `<Headline size='h3'/>`.
It will allows to remove the `'fontWeight: '700'` in styling.

Cf: https://www.figma.com/file/MD0xdHA7E93GKAwkSRpduK/SW-mobile-app---Design-Kit?node-id=23-5018&t=3Jd05KpM9L1wzdPF-4